source/modulo_components/component_descriptions/modulo_component.json

```
@@ -8,12 +8,20 @@
   "registration":  "modulo_components::Component",
   "inherits": "",
   "parameters": [
+    {
+      "display_name": "Rate",
+      "description": "The frequency in Hertz for all periodic callbacks (this parameter overrides the period parameter)",
+      "parameter_name": "rate",
+      "parameter_type": "int",
+      "default_value": "10"
+    },
     {
       "display_name": "Period",
       "description": "The time interval in seconds for all periodic callbacks",
```

Instead, you can put the detailed description here for reference
```suggestion
      "description": "The time interval in seconds for all periodic callbacks. This parameter is deprecated and will be removed in the next major release. The rate parameter takes precedence and overrides the component period.",
```